python/lsst/analysis/ap/zooniverseCutouts.py

```diff
@@ -50,33 +55,57 @@ class ZooniverseCutoutsConfig(pexConfig.Config):
         dtype=int,
         default=30,
     )
-    urlRoot = pexConfig.Field(
+    url_root = pexConfig.Field(
         doc="URL that the resulting images will be served to Zooniverse from, for the manifest file. "
             "If not set, no manifest file will be written.",
         dtype=str,
         default=None,
         optional=True,
     )
-    diffImageType = pexConfig.Field(
+    diff_image_type = pexConfig.Field(
         doc="Dataset type of template and difference image to use for cutouts; "
             "Will have '_templateExp' and '_differenceExp' appended for butler.get(), respectively.",
         dtype=str,
         default="deepDiff",
     )
-    addMetadata = pexConfig.Field(
+    add_metadata = pexConfig.Field(
         doc="Annotate the cutouts with catalog metadata, including coordinates, fluxes, flags, etc.",
         dtype=bool,
         default=False
     )
+    chunk_size = pexConfig.Field(
+        doc="Chunk up files into subdirectories, with at most this many files per directory."
+            " None (default) means write all the files to one `images/` directory.",
+        dtype=int,
+        default=None,
+        optional=True
+    )
 
 
 class ZooniverseCutoutsTask(lsst.pipe.base.Task):
-    """Generate cutouts and a manifest for upload to a Zooniverse project."""
+    """Generate cutouts and a manifest for upload to a Zooniverse project.
 
+    Parameters
+    ----------
+    output_path : `str`
+        The path to write the output to; manifest goes here, while the
+        images themselves go into ``output_path/images/``.
+    """
     ConfigClass = ZooniverseCutoutsConfig
     _DefaultName = "zooniverseCutouts"
 
-    def run(self, data, butler, outputPath):
+    def __init__(self, *, output_path, **kwargs):
+        super().__init__(**kwargs)
+        self._output_path = output_path
+        self.cutout_path = CutoutPath(output_path, chunk_size=self.config.chunk_size)
+
+    def _reduce_kwargs(self):
+        # to allow pickling of this Task
+        kwargs = super()._reduce_kwargs()
+        kwargs["output_path"] = self._output_path
+        return kwargs
```

Comment:
See the docstring for `pipe.base.task.Task._reduce_kwargs()`. I think it's if you have any args/kwargs beyond the default ones for `Task`.

Reply:
Just fell down a rabbit hole looking for them, failing to find them [here](https://pipelines.lsst.io/py-api/lsst.pipe.base.Task.html#lsst.pipe.base.Task) (because it's an `_` method?), reading the code, getting confused by random bits of that codebase, uggg... Should that method have docs there, do you think? I don't know whether to expect private methods to have docs listed there or not, but I bet you know!

Reply:
Here's the code link: https://github.com/lsst/pipe_base/blob/main/python/lsst/pipe/base/task.py#L414

No, private methods shouldn't have docs built, in general.

See this slack thread, including my "is this requirement documented anywhere?" question, which was not answered:
https://lsstc.slack.com/archives/C01FBUGM2CV/p1670637087157659